libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3601,6 +3602,30 @@ void GCS_MAVLINK::send_matternet_FTS_version(void) const
     }
 }
 
+/*
+  send active rangefinders' version strings
+ */
+void GCS_MAVLINK::send_rangefinder_versions(void) const
+{
+    RangeFinder *rangefinder = RangeFinder::get_singleton();
+    if (rangefinder == nullptr) {
+        return;
+    }
+    char rangefinder_version[80];
+    for (uint8_t i = 0; i < RANGEFINDER_MAX_INSTANCES; i++) {
+        AP_RangeFinder_Backend *sensor = rangefinder->get_backend(i);
+        if (sensor == nullptr) {
+            continue;
+        }
+        sensor->get_version(rangefinder_version, 80);
```

Comment:
instead of 80, pass sizeof(rangefinder_version)

Also set the last index to 0 or '\0'  null terminator to be safe.

Reply:
snprintf and stncpy already append null to end of these strings. But can add in case.

Reply:
The caller has no clue what the function is using.

Reply:
Also note that strncpy is not safe. Here is the documentation:

"If count is reached before the entire string src was copied, the resulting character array is not null-terminated."

Reply:
Fixed!